src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -352,7 +353,13 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
     <Box data-test="orderPayment">
       <OrderRouteContainer
         currentStep="Payment"
-        steps={order.mode === "OFFER" ? offerFlowSteps : buyNowFlowSteps}
+        steps={
+          order.mode === "OFFER"
+            ? offerFlowSteps
+            : order.source === "private_sale"
+            ? privateFlowSteps
+            : buyNowFlowSteps
+        }
```

Comment:
nonblocking suggestion: nested ternaries like this tend to always be super hard to reason about later on by devs who didn't write the code (and are bug prone / tend to be unwound during debugging, similar to implicit returns in react components. Best to just avoid it in favor of more boring `if` statements or a switch.

```tsx
let steps
if (offerMode === "OFFER") {
  steps = offerFlowSteps
} else {
  if (order.soruce === 'private_sale') {
    steps = privateFlowSteps
  } else {
    steps = buyNowSteps
  }
}
```
or the switch form, which is a bit clever but also quite clear:
```tsx
const steps = (() => {
  switch (true) {
    case offer.mode === "OFFER": 
      return offerFlowSteps
    case order.source === 'private_sale':
      return privateFlowSteps
    default:
      return buyNowFlowSteps
  }
})()
```